script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -352,8 +352,8 @@ Also, it's weird, but... when Noel-sensei describes Senpai as a monster, it does
 -- 「あ、埋葬機関っていうのは<聖堂教会|わたしたち>にとって、そうね……教会の中の治外法権、権力に近寄れないかわりに権力に縛られない、動く武器庫みたいなものと思って。
 "Oh, by the way, the Burial Agency is...
  Hmm, I guess you could think of it as this like... mobile arsenal.
- They're a part of the church that has complete diplomatic immunity.
- They don't have any real authority, but in exchange, they aren't really under the jurisdiction of anyone else's.
+ They're a part of the Church that has complete diplomatic immunity.
+ They don't have any real authority, but in exchange, they aren't really under anyone else's jurisdiction.
```

Comment:
Man... diplomatic immunity really doesn't sit right with me still to this day

just call it complete autonomy or something